src/main/java/com/google/cloud/tools/managedcloudsdk/command/AsyncByteConsumer.java

```diff
@@ -59,9 +59,10 @@
   @Override
   public void handleStream(final InputStream inputStream) {
     if (executorService.isShutdown()) {
-      throw new IllegalStateException("Cannot re-use " + this.getClass().getName());
+      throw new IllegalStateException("Cannot reuse " + this.getClass().getName());
     }
-    result.setFuture(executorService.submit(() -> consumeBytes(inputStream)));
+    ListenableFuture<String> submit = executorService.submit(() -> consumeBytes(inputStream));
```

Comment:
Any reason to expand this out?

Reply:
No semantic difference, but it makes refactoring, debugging, and updating easier. The more operations get crammed into a single statement the harder it becomes to read. In this case, it was difficult to demock without introducing local variables.